src/vivarium_csu_hypertension_sdc/model_specifications/test.yaml

```diff
@@ -49,3 +50,18 @@ configuration:
             by_age: False
             by_sex: False
             by_year: False
+        sample_history_observer:
+            sample_size: 1_000
+            path: /home/kate/vivarium_results/vivarium_csu_hypertension_sdc/test_sample_history.hdf
+
+    hypertension_treatment:
+        high_systolic_blood_pressure_measurement:
+            error_sd: 6
+        therapeutic_inertia:
+            mean: 0.136
+            sd: 0.0136
+        adverse_events:
+            mean: 0.1373333333
+            sd: 0.03069563849
+        followup_visit_interval:
+            weeks: 12
```

Comment:
I would flatten this key (e.g. just followup_visit_interval: value) and specify 90 days.  All other durations in our config are specified in terms of days and not given a key that describes their units.